packages/components/src/QualityBar/QualityBar.component.js

```diff
@@ -30,116 +73,82 @@ export function getQualityPercentagesRounded(invalid, empty, valid, na, placehol
 	const total = invalid + empty + valid + na + placeholder;
 
 	sumValues = (invalid * multiplier) / total;
-	const invalidRounded = Math.round(sumValues - sumRounded) / digitMultiplier;
+	output.invalid = Math.round(sumValues - sumRounded) / digitMultiplier;
 	sumRounded = Math.round(sumValues);
 
 	sumValues += (empty * multiplier) / total;
-	const emptyRounded = Math.round(sumValues - sumRounded) / digitMultiplier;
+	output.empty = Math.round(sumValues - sumRounded) / digitMultiplier;
 	sumRounded = Math.round(sumValues);
 
 	sumValues += (valid * multiplier) / total;
-	const validRounded = Math.round(sumValues - sumRounded) / digitMultiplier;
+	output.valid = Math.round(sumValues - sumRounded) / digitMultiplier;
 	sumRounded = Math.round(sumValues);
 
 	sumValues += (na * multiplier) / total;
-	const naRounded = Math.round(sumValues - sumRounded) / digitMultiplier;
+	output.na = Math.round(sumValues - sumRounded) / digitMultiplier;
 
 	sumValues += (placeholder * multiplier) / total;
-	const placeholderRounded = Math.round(sumValues - sumRounded) / digitMultiplier;
+	output.placeholder = Math.round(sumValues - sumRounded) / digitMultiplier;
 
-	return [invalidRounded, emptyRounded, validRounded, naRounded, placeholderRounded];
+	return output;
 }
 
-export function QualityBar({
-	digits = 1,
-	empty = 0,
-	getDataFeature,
-	invalid = 0,
-	na = 0,
-	placeholder = 0,
-	onClick,
-	valid = 0,
-}) {
-	const { t } = useTranslation(I18N_DOMAIN_COMPONENTS);
-
-	const [
-		invalidPercentage,
-		emptyPercentage,
-		validPercentage,
-		naPercentage,
-		placeholderPercentage,
-	] = getQualityPercentagesRounded(invalid, empty, valid, na, placeholder, digits);
+function SplitQualityBar({ empty, getDataFeature, invalid, na, onClick, valid, percentages }) {
```

Comment:
This one is used to show each component separately (see https://www.figma.com/file/4OL9bJO9KCX6DfMuHRgvLq/DQ-rules---M1?node-id=1035%3A46817)